### PR TITLE
Adding error codes to docker/opts package.

### PR DESCRIPTION
**Pull request by brahmaroutu**

### CODE REVIEW

File: opts/ip_test.go [hunk 1]
```diff
@@ -48,7 +50,7 @@ func TestIpOptSetInvalidVal(t *testing.T) {
 	invalidIP := "invalid ip"
 	expectedError := "invalid ip is not an ip address"
```
Comment (duglin) [inline opts/ip_test.go]: I wonder if this should be:
```
expectedError := derr.ErrorCodeInvalidIPFormat.Message
```

Comment (brahmaroutu) [inline opts/ip_test.go]: This is not as simple, by design derr.ErrorCodeInvalidIPFormat.Message gives you raw message, we will have to call the function with args.
The best is to go through phase II to eliminate static strings.

File: errors/opts.go [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+	ErrorCodeInvalidStream = errcode.Register(errGroup, errcode.ErrorDescriptor{
+		Value:          "INVALIDSTREAM",
+		Message:        "valid streams are STDIN, STDOUT and STDERR",
+		Description:    "The specified stream is invalid",
+		HTTPStatusCode: http.StatusInternalServerError,
+	})
+
+	// ErrorCodeInvalidVolPath is generated when volume path is incorrectly formatted.
+	ErrorCodeInvalidVolPath = errcode.Register(errGroup, errcode.ErrorDescriptor{
+		Value:          "INVALIDVOLUMEPATH",
```
Comment (duglin) [inline errors/opts.go]: should be INVALIDVOLPATH

File: errors/opts.go [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+		Message:        "bad format for path: %s",
+		Description:    "The specified path for the volume is incorrectly formatted",
+		HTTPStatusCode: http.StatusInternalServerError,
+	})
+
+	// ErrorCodeInvalidVolMode is generated when mode specified for the volume is incorrect.
+	ErrorCodeInvalidVolMode = errcode.Register(errGroup, errcode.ErrorDescriptor{
+		Value:          "INVALIDVOLUMEMODE",
```
Comment (duglin) [inline errors/opts.go]: s/VOLUME/VOL/

File: opts/ip_test.go (hunk 1 above)
Comment (duglin) [inline opts/ip_test.go]: Ah, I didn't realize it had a substitution - never mind then.

**Comment by LK4D4** [comment]
@brahmaroutu @duglin are we agree that we should close all such PRs for now?

**Pull request closed without merging**